server/pkg/analytics/analytics.go

```diff
@@ -59,17 +62,29 @@ func UpdateAnalyticsData() error {
 	if err != nil {
 		return fmt.Errorf("Error while setting up NewClient, err: %s", err)
 	}
-	response, err := svc.Data.Ga.Get(viewID, startDate, endDate, metrics).Dimensions(dimensions).Do()
+	response, err := svc.Data.Ga.Get(viewID, startDate, endDate, metrics).Dimensions(dimensions).Filters(filters).Do()
 	if err != nil {
 		return fmt.Errorf("Error while getting response, err: %s", err)
 	}
+	var sum int64
 	GAResponse := response.Rows
 	for i := range GAResponse {
-		object := GAResponseJSON{
-			Label: GAResponse[i][0],
-			Count: GAResponse[i][1],
+		if GAResponse[i][0] != "pod-delete-sa1xml" && GAResponse[i][0] != "pod-delete-s3onwz" && GAResponse[i][0] != "pod-delete-g85e2f" {
+			object := GAResponseJSON{
+				Label: GAResponse[i][0],
+				Count: GAResponse[i][1],
+			}
+			if GAResponse[i][0] != "Chaos-Operator" {
+				count,_ := strconv.ParseInt(object.Count, 10, 64)
+				sum = sum + count
+			}
+			GAResponseJSONObject = append(GAResponseJSONObject, object)
 		}
-		GAResponseJSONObject = append(GAResponseJSONObject, object)
 	}
+	object:= GAResponseJSON{
+		Label: "Total-Count",
+		Count: strconv.FormatInt(sum, 10),
```

Comment:
Here also some hardcoded number?

Reply:
It is specified as a parameter to convert to string. Required in function prototype. Have put in a constant instead.